src/System.Private.CoreLib/src/System/Threading/Thread.cs

```diff
@@ -145,6 +139,8 @@ internal sealed class Thread : RuntimeThread
         internal static CultureInfo m_CurrentCulture;
         [ThreadStatic]
         internal static CultureInfo m_CurrentUICulture;
+        [ThreadStatic]
```

Comment:
Nit: The prefix for thread statics should `t_`. 

`System.Threading.Thread` in CoreLib is left-over from full framework. It may be nice to do this in `RuntimeThread.cs` and get rid of `Thread.CurrentThread` in this file. (The FCall can still stay in this file.)

Reply:
> It may be nice to do this in RuntimeThread.cs and get rid of Thread.CurrentThread in this file. 

'Thread' is less accessible than 'RuntimeThread' but if it returned 'RuntimeThread' instead then that returns other errors so more properties would need to be moved up e.g.
```csharp
'RuntimeThread' does not contain a definition for 'ExecutionContext'
```

Reply:
What file is that? Most places should be covered by `using Thread = Internal.Runtime.Augments.RuntimeThread;` already.

Reply:
Ahh, I can add it then

Reply:
They are also places that use properties that are on `Thread` but not on `RuntimeThread`

Reply:
Changed to `t_` anyway 😄